### PR TITLE
Chelsea

### PR DESCRIPTION
**Pull request by XWangCS**
Chelsea's pull request

### CODE REVIEW

**Comment by tkevans** [comment]
Hi Chelsea! @XWangCS nice job! Thank you for writing the json data file! Before I merge your branch, can you delete the file "xcschememanagement.plist"? This file is not necessary for the project. Thanks!

File: Menu/MenuChoice.xcodeproj/xcuserdata/chelseaxie.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (HackBL) [inline Menu/MenuChoice.xcodeproj/xcuserdata/chelseaxie.xcuserdatad/xcschemes/xcschememanagement.plist:1]: This is not really useful, we could delete this file

**Comment by XWangCS** [comment]
Cannot find this file, so cannot delete it.

**Comment by XWangCS** [comment]
I will have a new branch for this changed json file.

**Pull request merged**